test/replicate.spec.js

```diff
@@ -110,6 +125,10 @@ Object.keys(testAPIs).forEach((IPFS) => {
         log2 = new Log(ipfs2, testIdentity2, { logId })
         input1 = new Log(ipfs1, testIdentity, { logId })
         input2 = new Log(ipfs2, testIdentity2, { logId })
+        const addr1 = await ipfs1.id()
+        const addr2 = await ipfs2.id()
+        await ipfs1.swarm.connect(addr2.addresses[0])
+        await ipfs2.swarm.connect(addr1.addresses[0])
```

Comment:
We could use `connectPeers()` from https://github.com/orbitdb/orbit-db-test-utils/blob/master/connect-peers.js here to fold these four lines to one.